.github/workflows/ci.yml

```diff
@@ -2,9 +2,15 @@ name: ci
 
 on:
   push:
+    inputs:
+      jdk-matrix:
```

Comment:
This needs to be in the `with` block below (`inputs` is used by the shared script: the `with` arguments here become `inputs` on the other side).